lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -96,7 +97,8 @@ def setUp(self):
         )
         CourseEnrollmentFactory.create(
             user=self.user,
-            course_id=self.course_id
+            course_id=self.course_id,
+            mode=CourseMode.HONOR,
```

Comment:
Similarly as above, testing audit doesn't really make sense. A test for verified would be reasonable, but I'm not entirely sure if it adds that much value for these tests since they should probably be agnostic as to the type of certificate.